DisCatSharp.ApplicationCommands/Checks/ApplicationCommandEqualityChecks.cs

```diff
@@ -101,21 +118,24 @@ internal static bool DeepEqual(DiscordApplicationCommand source, DiscordApplicat
 								minimalSubSubSourceOptions.Add(new DiscordApplicationCommandOption(
 									subSubOption.Name, subSubOption.Description, subSubOption.Type, subSubOption.Required ?? false,
 									subSubOption.Choices, null, subSubOption.ChannelTypes, subSubOption.AutoComplete ?? false,
-									subSubOption.MinimumValue, subSubOption.MaximumValue, null, null
+									subSubOption.MinimumValue, subSubOption.MaximumValue,
+									localizationEnabled ? subSubOption.NameLocalizations : null, localizationEnabled ? subSubOption.DescriptionLocalizations : null
 								));
 							}
 
 							minimalSubSourceOptions.Add(new DiscordApplicationCommandOption(
 								subOption.Name, subOption.Description, subOption.Type, null, null,
-								minimalSubSubSourceOptions, null, null, null, null, null, null
+								minimalSubSubSourceOptions, null, null, null, null,
```

Comment:
gotta admit the amount of null values being passed here is kinda nuts. Perhaps another constructor could be made available to help simplify things?